### PR TITLE
Version control updates with a few more git log options

### PR DESCRIPTION
**Pull request by jvsanchez28**

### CODE REVIEW

File: version-control/command-line/tutorial.md [hunk 1]
```diff
@@ -62,6 +62,33 @@ $ git log
...
+$ git log -p
+```
+
+If we want to see a few stats for each commit we can use the --stat command.
+
+```bash
+$ git log --stat
+```
+
+Even can do graphs with the --graph command. It will display an ASCII graph of the branch and merge history beside the log output.
```
Comment (KimberleyCook) [inline version-control/command-line/tutorial.md]: The first few words here I don't think make sense, maybe a word at the beginning is missing?

**Review by KimberleyCook — CHANGES_REQUESTED** [review]
Thank you very much for this addition @jvsanchez28, we really appreciate the help with these tutorials. I have just one little comment, once that is looked at I'm happy for it to be merged in :)

File: version-control/command-line/tutorial.md [hunk 2]
```diff
@@ -62,6 +62,33 @@ $ git log
 
 > Don't forget, you can hit `q` to quit out of a log if it's too long.
 
+By default, with no arguments, git log shows the most recent commits first.
+
+If we want to limit the number of log entries displayed we can use the a number as an option, such as -3 to show only the last three entries.
```
Comment (MarkCBJSS) [inline version-control/command-line/tutorial.md]: Small change here too: "...we can use **the a** number as an option..." should read as "we can use a number as an option"

Thanks for the improvements, Javier. Turning into a solid tutorial.

**Review by MarkCBJSS — CHANGES_REQUESTED** [review]

File: version-control/command-line/tutorial.md (hunk 2 above)
Comment (KimberleyCook) [reply version-control/command-line/tutorial.md]: Nice spot, thanks @MarkCBJSS

**Comment by jvsanchez28** [comment]
Thank you so much fro the review.
I have made a few changes to fix the errors :)

**Review by KimberleyCook — APPROVED** [review]
Sorry for the delayed re-review of this @jvsanchez28. Happy to merge this in, thanks for your work on it 💃

**Pull request merged**